assets/css/style.css

```diff
@@ -499,9 +505,59 @@ background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/s
     transform: scale(1.02);
     cursor: pointer
 }
+
+/* Queries */
+
 @media screen and (max-width: 500px) {
     /* to hide the scroll bars. Doesn't look good in browsers. Style it if you want   for mobile only */
     ::-webkit-scrollbar {
         display: none;
     }
 }
+
+/* @cobimr Responsive Update */
+@media screen and (max-width: 980px) {
+    /* Clean Blue Background on Mobile Device */
+    header.masthead {
+        background-position: initial
+    }
+    /* Fixed Responsive Main Banner*/
+    .masthead div.p-5 {
+        padding: 0 !important;
+    }
+    /* Fixed Discord Banner on Mobile Devices */
+    .discordheader {
+        padding-top: 10rem !important;
+    }
+    /* Better looking on About Discord Section by erasing innecesary padding*/
+    #about-discord div.p-5 {
+        padding: 0 !important;
+    }
+    /* Removing Discord Banner Parallax only for Mobile Devices */
+     section.paral{
+             background-attachment: unset;
+             margin: 0px;   
+     }
+
+    /* Better Image on mobile devices */
+    .testimonial-image {
+        width: 50% !important;
+    }
+    /* Desactivate Hover Animation on Mobile */
+    .testimonial {
+        transition: none !important;
+        margin: 1rem auto !important;
```

Comment:
looking pretty sweet with responsive changes! I have just enrolled in the class and saw your updates. Might consider removing !importants here and going with higher specificity rules --> https://css-tricks.com/specifics-on-css-specificity/. Is a good article, basically you could add div.testimonial or another identifier if you find your styles are not coming through currently. In general I have found !importants cause real headaches for devs with css, so that's why we try and simply avoid them by using bem naming convention or css in js approaches.... currently we do styled-components.

Reply:
Thank you very much for that tip Tim, I just read the CSS Tricks link you send me. I will take into consideration the good practice of specificity rules 🙏. I can fix the code in a moment and send another pull request but that would be in a few hours because in my country I have to sleep. No problem if I send a new pull request in a few hours?

Reply:
looks like it was already merged, but that sounds fine to me. Glad you found that useful